app/packages/looker/src/elements/common/tags.ts

```diff
@@ -426,3 +423,58 @@ const getFieldAndValue = (
 
   return [field, value, list];
 };
+
+const compareObjectArrays = (arr1, arr2) => {
```

Comment:
nit: can you use loadash for these?

Reply:
the lodash `isEqual` function is strict with the order of the array. These two helper functions only check if elements are the same, regardless of order.